cmsl1t/plotting/resolution.py

```diff
@@ -125,13 +143,25 @@ def __make_overlay(self, hists, fits, labels, ytitle, suffix=""):
             label_canvas()
 
             # Add a legend
-            legend = Legend(len(hists), header="Pile-up bin",
-                            topmargin=0.35, entryheight=0.035)
+            legend = Legend(
+                len(hists),
+                header=self.legend_title,
+                topmargin=0.35,
+                rightmargin=0.3,
+                leftmargin=0.7,
+                textsize=0.03,
+                entryheight=0.03,
+            )
+
             for hist, label in zip(hists, labels):
                 legend.AddEntry(hist, label)
             legend.SetBorderSize(0)
             legend.Draw()
 
+            ymax = 1.2 * hists[-1].GetMaximum()
```

Comment:
How is `hists` sorted?  Are we sure that the last one in the list will have the largest value?  Might need to do something like:
```python
ymax = 1.2 * max([hist.GetMaximum() for hist in hists])
```

Reply:
fixed